save-agent/src/linuxX64Main/kotlin/com/saveourtool/save/agent/AgentConfiguration.kt

```diff
@@ -33,12 +34,12 @@ data class AgentConfiguration(
     private val id: String,
     val backend: BackendConfig,
     val orchestratorUrl: String,
-    val heartbeat: HeartbeatConfig,
-    val requestTimeoutMillis: Long,
-    val retry: RetryConfig,
-    val debug: Boolean = false,
     val cliCommand: String,
-    val testSuitesDir: String,
+    val heartbeat: HeartbeatConfig = HeartbeatConfig(),
+    val requestTimeoutMillis: Long = 60000,
+    val retry: RetryConfig = RetryConfig(),
+    val debug: Boolean = false,
+    val testSuitesDir: String = ".",
```

Comment:
I'll remove setting of both `CLI_COMMAND` and `TEST_SUITES_DIR` from orchestrator. Since agent now downloads tests on its own, it can decide where to store them, unless specifically overridden. Same goes for cli command, as you've noted in another comment